src/test/java/com/VetApp/PurrgentCare/services/AccountServiceImplementationTest.java

```diff
@@ -215,4 +231,43 @@ public void toggleAccount_whenAccountNotExists_throwEntityNotFoundException() {
         then(exception.getMessage()).contains(String.valueOf(accountId));
 
     }
+
+    // Test associatePeople when account exists
+        //This method adds a list of personIds to an account
+        //
+    @Test
+    public void associatePeople_whenAccountExists_returnAssociatedPeople() {
+        // given
+        // mock associatepeoplewithaccountrequest
+        // mock accountId
+        // mock List<PersonIds>
+
+        final var accountId = new Random().nextInt(1000);
+        final var personId = new Random().nextInt(1000);
+        final var peopleList = new ArrayList<Integer>(personId);
+        final AssociatePeopleWithAccountRequest dtoAssociatePeopleWithAccountRequest = new AssociatePeopleWithAccountRequest();
+        dtoAssociatePeopleWithAccountRequest.accountId = accountId;
+        dtoAssociatePeopleWithAccountRequest.personIds = peopleList;
+        final var person = new Person();
+        final var pet = new Pet();
+        final var account = new Account();
+        final var listOfPeople = new ArrayList<Person>(person);
+        final var accountResponse = new AccountResponse();
+        accountResponse.active = Boolean.TRUE;
+        accountResponse.dateCreated = new Date();
+        accountResponse.accountHolders = new ArrayList<Person>(person);
+        accountResponse.pets = new ArrayList<Pet>(pet);
+
+        given(mockAccountRepository.findById(accountId))
+                .willReturn(Optional.of(account));
+        given(mockPersonRepository.findAllById((peopleList))
+                .willReturn(Optional.of(personId)));
```

Comment:
You need additional mocking.
In the service implementation...
After we get the people and account from the repo...
We edit the account then save it...
We then convert the response from the save method using the modelMapper...
Where did we mock the modelMapper's activity?
i.e. `AccountResponse accountResponse = mapper.map(updatedAccount, AccountResponse.class);` --> where is this behavior mocked in our test?

Reply:
See commit here for what I mean...
https://github.com/JesseMattson/PurrgentCare/pull/61/commits/bd58f912317729c7684ec747ceaa7b32c54b4c44